apps/site/lib/site/phone_number.ex

```diff
@@ -42,6 +42,27 @@ defmodule Site.PhoneNumber do
     end
   end
 
+  @doc """
+  Format read by screenreaders in a nicer manner
```

Comment:
Curious as to how phone numbers are read out with this change?

Reply:
:grin: For example for Boston area code `617` it should read  "six one seven" instead of "six hundred and seven"